src/test/java/vaskevich/assertions_test/HoffSearchTest.java

```diff
@@ -1,4 +1,17 @@
 package vaskevich.assertions_test;
 
+import com.codeborne.selenide.Condition;
+import org.junit.jupiter.api.Test;
+
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.open;
+
 public class HoffSearchTest {
+
+    @Test
+    public void searchHoffProduct(){
+        open("https://hoff.ru/");
+        $("input[name='search']").setValue("гостиная");
+        $("h1.page-title").shouldHave(Condition.text("Гостиная"));
+    }
```

Comment:
Great job, but please change something here or add any comments in your code.

Reply:
Added commits actions in HoffSearchTest